src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -58,14 +58,14 @@ public void execute_invalidCommandFormat_throwsParseException() {
 
     @Test
     public void execute_commandExecutionError_throwsCommandException() {
-        String deleteCommand = "delete 9";
+        String deleteCommand = "people delete 9";
         assertCommandException(deleteCommand, MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
     }
 
     @Test
     public void execute_validCommand_success() throws Exception {
-        String listCommand = ListCommand.COMMAND_WORD;
-        assertCommandSuccess(listCommand, ListCommand.MESSAGE_SUCCESS, model);
+        String listCommand = "people " + PeopleListCommand.COMMAND_WORD;
```

Comment:
I think abit inconsistent because you use PEOPLE_COMMAND_TYPE for some tests but just the raw string "people" for other tests so can standardise